### DISCUSSION TITLE
About the projects.json file

### DISCUSSION THREAD

**Discussion by fade2gray** [alefragnani/vscode-project-manager#463] [Q&A] [answered]
![Capture](https://user-images.githubusercontent.com/1332009/106481527-6f522280-64a4-11eb-9ef0-912ce4a5e9ac.PNG)

Thanks for any info.

**Comment by alefragnani** [comment, marked as answer]
Hi @fade2gray,

Ok, let's go 😄 :

* `paths`: The idea came a bit before the VS Code team started working in **multi-root workspaces**, with the same purpose. In the end, the VS Code implementation was well received, and the attribute ended up _obsolete_. But, there is an open issue (https://github.com/alefragnani/vscode-project-manager/issues/121) asking to _use_ this attribute as a way to _open new VS Code windows with those paths_

* `group`: The idea is to give each project a _group/tag_, in a way to organize folders. It's directly associated with https://github.com/alefragnani/vscode-project-manager/issues/50, but it will be replaced to be an `array` instead of a string. More details about how I'm planning is commented here https://github.com/alefragnani/vscode-project-manager/issues/50#issuecomment-769053196

* `enabled`: It refers to the possibility to **Hide** projects, in the **Favorites** panel. It was originally requested https://github.com/alefragnani/vscode-project-manager/issues/154, contributed via this PR https://github.com/alefragnani/vscode-project-manager/pull/226. There is an open issue (https://github.com/alefragnani/vscode-project-manager/issues/368) to make some improvements on that.

Hope this helps

**Reply by fade2gray** [reply]
@alefragnani  Thank you for the explanations. 👍

While I have your attention, can I point you [here](https://github.com/alefragnani/vscode-project-manager/issues/462)?

**Comment by vilija19** [comment]
> Hi @fade2gray,
> 
> Ok, let's go 😄 :
> 
> * `paths`: The idea came a bit before the VS Code team started working in **multi-root workspaces**, with the same purpose. In the end, the VS Code implementation was well received, and the attribute ended up _obsolete_. But, there is an open issue ([Option to open folders in paths as new windows #121](https://github.com/alefragnani/vscode-project-manager/issues/121)) asking to _use_ this attribute as a way to _open new VS Code windows with those paths_
> * `group`: The idea is to give each project a _group/tag_, in a way to organize folders. It's directly associated with [Contextual Structure for Projects #50](https://github.com/alefragnani/vscode-project-manager/issues/50), but it will be replaced to be an `array` instead of a string. More details about how I'm planning is commented here [Contextual Structure for Projects #50 (comment)](https://github.com/alefragnani/vscode-project-manager/issues/50#issuecomment-769053196)
> * `enabled`: It refers to the possibility to **Hide** projects, in the **Favorites** panel. It was originally requested [Hide Project #154](https://github.com/alefragnani/vscode-project-manager/issues/154), contributed via this PR [Add ability to temporarily disable individual projects (#154) #226](https://github.com/alefragnani/vscode-project-manager/pull/226). There is an open issue ([[FEATURE] - Improve "Disabled Projects" experience #368](https://github.com/alefragnani/vscode-project-manager/issues/368)) to make some improvements on that.
> 
> Hope this helps



Is it still possible to use paths to work with multiroot projects?
If so, would you give some examples of paths?
VScode multi-root workspaces are not so comfortable to work in, because there is no way to open the .code-workspace file at the start of a session automatically.

**Reply by alefragnani** [reply]
Hi @vilija19 ,

No, it is not possible. That attribute isn't being used yet, and would probably be removed in future releases.

I'm not sure what you mean about _there is no way to open a .code-workspace file at the start of a session automatically_. What s your use case?
